spec/support/idv_examples/max_attempts.rb

```diff
@@ -0,0 +1,100 @@
+shared_examples 'idv max step attempts' do |sp|
+  it 'allows 3 attempts in 24 hours', :email do
+    visit_idp_from_sp_with_loa3(sp)
+    user = register_user
+
+    max_attempts_less_one.times do
+      visit verify_session_path
+      fill_out_idv_form_fail
+      click_idv_continue
+
+      expect(current_path).to eq verify_session_result_path
+    end
+
+    user.reload
+    expect(user.idv_attempted_at).to_not be_nil
+
+    fill_out_idv_form_fail
+    click_idv_continue
+
+    expect(page).to have_css('.alert-error', text: t('idv.modal.sessions.heading'))
+
+    visit_idp_from_sp_with_loa3(sp)
+    expect(page).to have_content(t('idv.messages.hardfail'))
+    expect(current_url).to eq verify_fail_url
+
+    visit verify_session_path
+    expect(page).to have_content(t('idv.errors.hardfail'))
+    expect(current_url).to eq verify_fail_url
+
+    user.reload
+    expect(user.idv_attempted_at).to_not be_nil
+  end
+
+  scenario 'profile shows failure flash message after max attempts', :email do
```

Comment:
Same thing here as above. We only had examples like this for the financial step, though an example for the profile step is tied up in the test above.